Death added.

nickums224 opened the pull request:
Added the ability for minor to die, and then code breaks when health reaches 0.

nickums224 committed:
Death added.

Added the ability for minor to die, and then code breaks when health reaches 0.

jzmiller1 commented:
So you're adding health to the Wife but not to the Miner?  Also, the only place you try to adjust the health is in the mining state.  There's also an error in the health adjustment:

        if miner.thirst >= 2:
            miner.health - 30

should  probably be:

        if miner.thirst >= 2:
            miner.health -= 30

but that will only work if the miner has a health attribute.  Need to do a little more work on this.  If you have specific questions let me know.

nickums224 committed:
DeathState

When one/both of miners' health reaches 0 they die and entire a loop in the death state.

jzmiller1 commented:
There have been a lot of changes in the master repository.  You'll need to update your fork of master and rebase your changes onto it.
